opentech/apply/funds/templates/funds/application_base.html

```diff
@@ -32,9 +32,13 @@ <h3>{% blocktrans %}Sorry this {{ page|verbose_name }} is not accepting applicat
 
             {% for field in form %}
               {% if field.field %}
-                {% include "forms/includes/field.html" %}
+                <div class="field-group-{{ field.field.group }}">
```

Comment:
Would it be possible to add this class to the wrapping div in field.html? So we avoid to many levels of divs.

Reply:
Yeah, PR updated.